di-ipv-ticf-stub/lambdas/src/management/handlers/managementHandler.ts

```diff
@@ -11,7 +11,7 @@ export async function handler(
   if (!userId) {
     return buildApiResponse({ errorMessage: "Missing userId." }, 400);
   }
-
+  const statusCode = event.pathParameters?.statusCode;
```

Comment:
If you default the status code here it'll make the rest of the code neater, and I think it's the right place for it too (the fact it defaults to 200 is a property of the management API, not the storage service).

i.e. `const statusCode = event.pathParameters?.statusCode ?? 200;`

Reply:
done...bring it up to api level.